gc/gc-iceberg/src/main/java/org/apache/iceberg/spark/procedures/BaseGcProcedure.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.iceberg.spark.procedures;
+
+import org.apache.spark.sql.connector.catalog.TableCatalog;
+
+/**
+ * This class is a "bridge" between the Iceberg-Spark package and the Nessie package, because {@link
```

Comment:
nit: too breat the doc line a bit earlier

Reply:
I think it is spotless doing ;)